lib/msal-angular/README.md

```diff
@@ -91,6 +91,7 @@ All documentation for MSAL Angular v1 can be found [here](https://github.com/Azu
 * [Angular v11](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-angular-v2-samples/angular11-sample-app)
 * [Angular v11 B2C Sample](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-angular-v2-samples/angular11-b2c-sample)
 
+Note: Angular Universal is also supported. Please see the [Angular v11 sample](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-angular-v2-samples/angular11-sample-app) for notes on implementation.
```

Comment:
I think this would go well under advanced usage topics.

Reply:
Moved to its own doc.